src/main/java/org/isf/patient/rest/PatientController.java

```diff
@@ -232,45 +231,46 @@ public ResponseEntity<PatientDTO> getPatientAll(@RequestParam int code) throws O
 
 	@GetMapping(value = "/patients/nextcode", produces = MediaType.APPLICATION_JSON_VALUE)
 	public ResponseEntity<Integer> getPatientNextCode() throws OHServiceException {
-		LOGGER.info("get patient next code");
+		LOGGER.info("Get patient next code.");
 		int nextCode = patientManager.getNextPatientCode();
 		return ResponseEntity.ok(nextCode);
 	}
 
 	@DeleteMapping(value = "/patients/{code}", produces = MediaType.APPLICATION_JSON_VALUE)
 	public ResponseEntity<Boolean> deletePatient(@PathVariable int code) throws OHServiceException {
-		LOGGER.info("Delete patient code: {}", code);
+		LOGGER.info("Delete patient code: `{}`.", code);
```

Comment:
is there a reason why backtick instead of single quote here?

Reply:
fixed; thanks